callstack/repack
# feat: add repack bootstrap code to await chunk loading

### Summary

Adds option to use `await ChunkManager.loadChunk(...)` to await the execution of the additional code.

### Test plan

1. Run TesterApp
2. It should work as before the change
3. Use `await ChunkManager.loadChunk(...)` to run code once the chunk has been evaluated
